src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -250,6 +252,7 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             [$operation['method']],
             $operation['condition'] ?? ''
         );
+        $name = RouteNameGenerator::generate($operationName, $resourceShortName, $operationType);
```

Comment:
is this really needed?

Reply:
No sorry, was there from previous workings related to the 'other outstanding issue' I mentioned - I forgot to remove it.